src/app/core/building/building.component.html

```diff
@@ -511,7 +511,7 @@ <h4>3. 指定容積率と比較し、小さい方を採用</h4>
                 <tbody>
                   <tr>
                     <td>容積率の上限</td>
-                    <td class='text-right'>{{result?.buildingSite?.floorAreaRatioLimit}}%</td>
+                    <td class='text-right'>{{Math.min(result?.buildingSite?.floorAreaRatioLimit * 100, blueprint.floorAreaRatio)}}%</td>
```

Comment:
@thanhtetaung  Please implement here to work.

Reply:
@k-kin this kind of logic should not be on frontend side. Should return from backend. As I remember Hla Min already told about this previously.

Reply:
@thanhtetaung  All right, I add it to response.

Reply:
I changed here, so close this conversation